frontend/packages/core/src/dialog.tsx

```diff
@@ -1,27 +1,61 @@
-import React from "react";
-import MuiDialog from "@material-ui/core/Dialog";
-import DialogActions from "@material-ui/core/DialogActions";
-import DialogContent from "@material-ui/core/DialogContent";
-import DialogTitle from "@material-ui/core/DialogTitle";
-import Typography from "@material-ui/core/Typography";
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { DialogProps as MuiDialogProps } from "@material-ui/core";
+import {
+  Dialog as MuiDialog,
+  DialogActions as MuiDialogActions,
+  DialogContent as MuiDialogContent,
+  DialogTitle as MuiDialogTitle,
+  IconButton as MuiIconButton,
+} from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
 
-export interface DialogProps {
+const DialogTitle = styled(MuiDialogTitle)({
+  display: "flex",
+  justifyContent: "space-between",
+  fontSize: "16px",
+  padding: "16px 16px 0 16px",
+  fontWeight: 400,
+  color: "#000000",
+});
+
+const IconButton = styled(MuiIconButton)({
+  height: "16px",
+  width: "16px",
+  padding: "0",
+  color: "#000000",
+});
+
+const DialogContent = styled(MuiDialogContent)({
+  padding: "8px 16px",
+  fontSize: "12px",
+  fontWeight: 400,
+  color: "rgba(13, 16, 48, 0.6)",
+});
+
+const DialogActions = styled(MuiDialogActions)({
+  borderTop: "1px solid rgba(13, 16, 48, 0.12)",
```

Comment:
hmm should we use `<Divider>` or just add the border to actions like this? (asking for `<Accordion>`)

Reply:
I think for now I'll leave this as a border. It makes it slightly easier for purposes of the children. e.g. if we used a divider we would need to wrap the DialogActions children to prepend the divider to them